extensions/kind/package.json

```diff
@@ -51,7 +51,9 @@
     "watch": "tsc -w"
   },
   "dependencies": {
-    "@tmpwip/extension-api": "^0.0.1"
+    "@tmpwip/extension-api": "^0.0.1",
+    "@terascope/fetch-github-release": "^0.8.7",
```

Comment:
I would avoid this dependency if possible. It seems to depend on too many extra dependencies 

Also when checking PR with extensions, please also check a production bibary